clang/tools/dpct/cmake/dpct.cmake

```diff
@@ -78,3 +78,15 @@ endmacro()
 # Always set SYCL_HAS_FP16 to true to assume SYCL device to support float16
 message("dpct.cmake: SYCL_HAS_FP16 is set true by default.")
 set(SYCL_HAS_FP16 TRUE)
+
+# Replaces cuda_compile with it's equivalent SYCL repr
```

Comment:
L82 comment is not correct, it should show the fun of DPCT_HELPER_SYCL_COMPILE

Reply:
That's correct! Updated the comment to the following

> # Return the list of object file paths generated for the given SYCL source files